docs/INSTALL.md

```diff
@@ -130,35 +130,20 @@ tools/unix/build_omim.sh -d help
 
 The generated binaries appear in `../omim-build-<buildtype>`.
 
-Run `OMaps` binary from `../omim-build-<buildtype>`, for example, for release:
+A desktop app binary is `OMaps`. To run e.g. a release version:
 
 _Linux:_
 
 ```bash
 ../omim-build-release/OMaps -data_path ./data
 ```
 
-or create `data` symlink in build dir to `organicmaps/data` directory and run
```

Comment:
Please add a comment that the binary expects a `data` folder in the current working dir if -data_path was not specified. Btw, let's rename it to -data.

Reply:
Maybe it would be better to just reference to `OMaps -help`?

BTW looking at it I discovered `OMaps -lang ru` option which looks like a better way than passing an env var (as no issues with misc env vars overriding each other).

Reply:
re renaming `-data_path` to `-data`: it looks like the `-data_path` is a conventional option used by many tools and tests, e.g. generator_tool, run_desktop_tests.py etc.

I think its better to keep it consistent across different tools.

Reply:
It would be great to print -help options in the Readme.

It's easy to grep and change it everywhere ) together with other _path options. But ok, let's leave it for another time.